style.css

```diff
@@ -318,7 +349,43 @@ button:focus {
   box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
 }
 
-.disk-br-about { display: none; }
+.disk-br-about {
+  display: none;
+}
+
+.project-model {
+  display: none;
+  background: #c1c7d064;
+  backdrop-filter: blur(2px);
+  align-items: center;
+}
+
+.main-view {
+  width: 100%;
+  padding: 60px 16px;
+}
+
+.project-view {
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.main-view .details {
+  display: flex;
+  flex-direction: column;
+}
+
+.project-model,
+.menu-modal {
```

Comment:
- [ ] Nice job so far, But you just have a little fixing to make. Your modal desktop view is taking whole screen height making it difficult to see all the content in the modal, kindly fix this.
   how it looks 👇 
      
![image](https://user-images.githubusercontent.com/66097371/161822718-f904e3b8-69b1-424f-8426-280ce76e1ec7.png)

Reply:
Thanks 👍😊